R-Interface/jasprcpp.cpp

```diff
@@ -71,6 +71,12 @@ static std::string				_R_HOME = "";
 
 bool shouldCrashSoon = false; //Simply here to allow a developer to force a crash
 
+// TODO: this probably won't work on windows
```

Comment:
Haha no, now you've broken all utf-8 in windows ;)

But the next R has UCRT as runtime and thus utf-8 as native encoding so once we switch to that we can strip all of this nonsense anyway.

Reply:
yeah, I figured, but I didn't want to spend a lot of time on that part before discussing the rest of the approach.